packages/editor/src/js/vue/components/personalized-blocks-list/personalized-blocks-list-component.js

```diff
@@ -1,6 +1,8 @@
 const Vue = require('vue/dist/vue.common');
 const axios = require('axios');
 const { getPersonalizedBlocks } = require('../../utils/apis');
+const debounce = require('lodash.debounce');
+const DEBOUNCE_DELAY_MS = 400;
```

Comment:
You could extract this variable in an utils file for inputs (if it doesn't exist, create a new one)